Sources/ProjectDescription/Entitlements.swift

```diff
@@ -9,6 +9,9 @@ public enum Entitlements: Codable, Equatable {
     /// A dictionary with the entitlements content. Tuist generates the .entitlements file at the generation time.
     case dictionary([String: Plist.Value])
 
+    /// A user defined xcconfig variable map to .entitlements file
+    case variable(String)
```

Comment:
Can you expand a little bit on the docs here? Maybe even with a small usage sample or why this is useful in the first place?